scico/typing.py

```diff
@@ -9,6 +9,12 @@
 
 from typing import Any, Tuple, Union
 
+try:
+    # available in python 3.10
+    from typing import EllipsisType  # type: ignore
+except ImportError:
+    EllipsisType = Any
```

Comment:
@Michael-T-McCann: why `EllipsisType = Any` rather than `EllipsisType = type(Ellipsis)`?

Reply:
No idea; I don't think I wrote this code.